k6/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/mdsol/xk6-mauth
```

Can you change the folder name from `k6` to `xk6-mauth`?
```suggestion
module github.com/mdsol/go-mauth-client/xk6-mauth
```